doc/grass_database.html

```diff
@@ -2,16 +2,17 @@
 
 A GRASS GIS Database is simply a set of directories and files
 with certain structure which GRASS GIS works efficiently with.
-Location is a directory with data related to
-one geographic location or a project.
-All data within one Location has the same cartographic projection.
-A Location contains Mapsets and each Mapset contains data related to
+Project is a directory with data related to
+one geographic location or a real-world project.
```

Comment:
```suggestion
one geographic location or a project.
```

I think you don't need "real-world" here. It doesn't add much clarity.